.github/workflows/ci.yml

```diff
@@ -339,7 +307,7 @@ jobs:
           name: x86_64-pc-windows-msvc-dll
           path: bound/kt/src/main/resources/
 
-      - name: Deploy Snapshot / Verify
+      - name: Build and Test Kotlin Project
```

Comment:
I'd probably leave something related to the Snapshot in this title, because we are creating a snapshot in our artifactory